test/navbar-components/navigationBarTest.js

```diff
@@ -379,10 +379,8 @@ describe('navigationBar tests', function () {
           this.app.client
             .activateURLMode()
             .waitForExist(urlBarIcon)
-
-            // TODO: Fix
             .getAttribute(urlBarIcon, 'class').then((classes) =>
-              classes.includes('fa-unlock') && classes.includes('insecure-color')
+              classes.includes('fa-unlock')
```

Comment:
I agree, yet I don't know how to add data-test-id change conditions, following [iconCssClasses](https://github.com/brave/browser-laptop/pull/9851/files#diff-51a114fecaac13cd41162f4c956e7508R54) on urlBarIcon.js. Could I ask you to do that?